packages/big-design/src/components/Lozenge/styled.tsx

```diff
@@ -0,0 +1,54 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { Box } from '../Box';
+
+import { LozengeProps } from './Lozenge';
+
+interface StyledLozengeProps extends Omit<LozengeProps, 'label' | 'tooltipContent'> {
+  hasTooltip?: boolean;
```

Comment:
might as well be mandatory, given your parent component is always providing it

optional props are more useful when the client is external (e.g. someone using BigDesign in their own app) as it saves them entering values when there's safe/expected defaults

given this component API isn't directly public facing we can have it more clearly state what it needs

then we don't the need the defaulting (line 53) below either